Resources/doc/events.rst

```diff
@@ -63,7 +63,11 @@ when its event is dispatched.
 Event Subscribers
 -----------------
 
-Use the ``doctrine_mongodb.odm.event_subscriber`` tag
+Implement ``Doctrine\Bundle\MongoDBBundle\EventSubscriber\EventSubscriberInterface``
```

Comment:
```suggestion
Implementing ``Doctrine\Bundle\MongoDBBundle\EventSubscriber\EventSubscriberInterface``
```

Reply:
Why replacing Use by Implement**ing** ?

Reply:
You can replace it with implement, but to be grammatically correct you will have to continue with "and autoconfiguration to automatically register"

Reply:
Ok, my explanation was incomplete. I was not telling to use autoconfiguration but that autoconfiguration will do the job. But you're right, I should explicitely tell to use autoconfigure.

Reply:
I was more trying on "to" vs "will" here, not sure if that came through

Reply:
Is it better now ?

Reply:
It sure is 👍